server/container_create_linux.go

```diff
@@ -799,6 +800,15 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 		return nil, err
 	}
 
+	if sb.Annotations()[crioann.UmaskAnnotation] != "" {
```

Comment:
we may want to do `if v, _ := ...; v != ""` to make sure the annotation isn't added

Reply:
@haircommander could you elaborate? did you mean we need to check the v is not black spaces?